### ISSUE TITLE
Area maps incorrectly to the screen after disconnecting one monitor in a multi-monitor scenario

### ISSUE THREAD

**Issue by Kyuunex** [OpenTabletDriver/OpenTabletDriver#1143]
## Description
If I boot up my PC with 2 monitors attached, later disconnect one, the area mapped on the screen will mess up.
Same if I boot up with 1 monitor and attach another. 
For example, the tablet area will map to both screens side by side.

doing `systemctl --user restart opentabletdriver` fixes the issue for the time being.
OTD GUI also requires a restart to refresh the list of displays in the "set to display".

would be nice if this was automated somehow

## System Information:

| Name             | Value |
| ---------------- | ----- |
| Operating System | Arch Linux |
| Software Version | rolling release, everything is always latest |
| Tablet           | XP-PEN G430 |

**Comment by InfinityGhost** [comment]
> rolling release, everything is always latest

Not necessarily true, still need the hash at least.

**Comment by Kyuunex** [comment]
> > rolling release, everything is always latest
> 
> Not necessarily true, still need the hash at least.

specifically of what you need a version of?

**Comment by Phyan** [comment]
Yeah, please do that, guys. Using more than one monitor that are constantly manually connected or using software like Spacedeck makes it impossible to comfortably use a graphics tablet, since there is no adequate mapping to the specific monitor that is used as the main workspace.

**Comment by gonX** [comment]
Still hoping to fix this, but the linked PR (#4017) was too janky, and knowing how much effort that took (and it still didn't succeed) I don't think it's realistic to look at this in time for the upcoming release, and may not be fixable before some substantial core changes are made.
